Resources/Prototypes/_LostParadise/Research/Arsenal.yml

```
@@ -35,4 +35,17 @@
+
+- type: technology
+  id: LPPCombatPlasmaUsage
+  name: lppresearch-combat-plasma-usage
+  icon:
+    sprite: _LostParadise/Objects/Weapons/Melee/plasma_sword.rsi
+    state: p_sword
+  discipline: Arsenal
+  tier: 3
+  cost: 25000
+  recipeUnlocks:
+  - LPPRecipePlasmaSword
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)